### PR TITLE
Unit2 tweaks

### PR DESCRIPTION
**Pull request by lewtun**
ReviewNB didn't do a good job on the diff - are you using `nbdev` by any chance to strip out metadata? I can easily run this too if needed :)

### CODE REVIEW

File: unit2/README.md [hunk 1]
```diff
@@ -8,46 +8,40 @@ Here are the steps for this unit:
 
 - Make sure you've [signed up for this course](https://huggingface.us17.list-manage.com/subscribe?u=7f57e683fa28b51bfc493d048&id=ef963b4162) so that you can be notified when new material is released
 - Read through the material below for an overview of the key ideas of this unit
-- Check out the _**FINE TUNING AND GUIDANCE NOTEBOOK TODO LINK**_ to fine-tune an existing diffusion model on a new dataset using the 🤗 Diffusers library
```
Comment (lewtun) [inline unit2/README.md:11]: I removed the references to conditioning for now - let me know if they should be re-instated

File: unit2/README.md [hunk 2]
```diff
@@ -8,46 +8,40 @@ Here are the steps for this unit:
...
 ## Project Time
 
-Create a custom pipeline using some or all of the ideas covered in this unit and share it with the community
+The best way to learn is by doing! Fine-tune your own diffusion model on a new image dataset and create a Gradio demo that can generate images from it. Don't forget to share your work with us on Discord, Twitter or elsewhere!
 
 ## Some Additional Resources
  
-GLIDE (text conditioned diffusion model) TODO link
-
-Thomas' example TODO link
```
Comment (lewtun) [inline unit2/README.md:49]: I removed this for now, but we can add it whenever you want (I wasn't sure which Thomas you're referring to :))

**Comment by johnowhitaker** [comment]
I should have cleaned the notebook. Do you want to do that and then I'll merge, so at least going forward we'll have nicer diffs?

File: unit2/README.md (hunk 1 above)
Comment (johnowhitaker) [reply unit2/README.md:11]: I've done a small illustrative notebook and will have some extra info on conditioning to add to the readme later today.

**Pull request merged**